src/mongocxx/test/versioned_api.cpp

```diff
@@ -28,15 +28,8 @@ using namespace mongocxx;
 // clang-format off
 
 static bool is_server_v5_or_newer() {
-    using namespace bsoncxx::builder::basic;
-    client cl{uri{}};
-    auto status = cl["admin"].run_command(make_document(kvp("serverStatus", 1)));
-    auto version = bsoncxx::string::to_string(status["version"].get_string().value);
-    auto dot_pos = version.find('.');
-    REQUIRE(dot_pos != version.npos);
-    auto major_digit_str = version.substr(0, dot_pos);
-    auto major_version = std::stoul(major_digit_str);
-    return major_version >= 5;
+    auto wire_version = mongocxx::test_util::get_max_wire_version(client{uri{}});
```

Comment:
Use `client{uri{}, test_util::add_test_server_api()}` to apply the necessary default versioned API test on the `test_versioned_api` task.